notebooks/249-oneformer-segmentation/249-oneformer-segmentation.ipynb

```diff
@@ -14,7 +14,7 @@
    "id": "b6dbb8aa-c070-4e7d-8b7a-64588e7f0da4",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q "transformers&gt;=4.26.0" gradio
</pre><p>please restore dependencies installation and add nncf</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I removed the rest because they're already present in `requirements.txt`. As you can see in the CI, there are no errors of missing packages for this notebook. `nncf` is also present in  `requirements.txt`.

Reply:
the current strategy to make notebooks self-contained without usage requirements.txt (they stayed for backward compatibility only and speedup our internal testing that is why notebook is not failed), to be able run on colab or other cloud environments